heartbeat/scheduler/scheduler.go

```diff
@@ -189,11 +190,24 @@ func (s *Scheduler) Add(sched Schedule, id string, entrypoint TaskFunc, jobType
 		debugf("Job '%s' started", id)
 		sj := newSchedJob(jobCtx, s, id, jobType, entrypoint)
 
-		lastRanAt := sj.run()
+		inMaintWin := false
+		for _, pmw := range pmws {
+			if pmw.IsActive(now) {
+				inMaintWin = true
+			}
+		}
+		var lastRanAt time.Time
+		if !inMaintWin {
+			lastRanAt = sj.run()
+		} else {
+			lastRanAt = now
```

Comment:
Can I suggest adding a `sync.Once` log message here with the rule that triggered the skip? I'm worried there won't be any visibility in these cases and that might be difficult to debug from a support perspective.